api-reference/10 UI Widgets/dxLookup/1 Configuration/itemCenteringEnabled.md

```diff
@@ -0,0 +1,13 @@
+---
+id: dxLookup.Options.itemCenteringEnabled
+type: Boolean
+default: false, true (Material)
+---
+---
+##### shortDescription
+Specifies whether to center the drop-down menu on the selected item. Applies only in [Material Design themes](/Documentation/Guide/Themes_and_Styles/Predefined_Themes/).
```

Comment:
I really don't understand the first sentence. Does it mean something like 'Specifies whether to align the drop-down menu on center of the selected item? Or did I get it wrong?

Reply:
```suggestion
Specifies whether to vertically align the drop-down menu so that the selected item is in its center. Applies only in [Material Design themes](/Documentation/Guide/Themes_and_Styles/Predefined_Themes/).
```